src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml

```diff
@@ -629,7 +628,12 @@
 									Style="{StaticResource MaterialRevealButtonStyle}"
 									Visibility="Collapsed" />
 						</Grid>
-					</Border>
+
+						<Border x:Name="BorderBorder"
```

Comment:
it is important, to comment, when you are using an esoteric setup
copy the comment i left on `BorderBorder` in `MaterialOutlinedTextBoxStyle`